src/TableBrowser.h

```diff
@@ -64,7 +65,8 @@ struct BrowseDataTableSettings
         if(stream.atEnd())
             return stream;
         stream >> object.plotXAxis;
-        stream >> object.plotYAxes;
+        stream >> object.plotY1Axes;
+        stream >> object.plotY2Axes;
```

Comment:
This is the old stream format, which we abandoned in 3.11, but the loading is still kept here for backwards compatibility, so it shouldn't be affected by this change. I suppose it would work simply by using object.plotY1Axes where object.plotXAxis was used.  

If it were possible, it would be cool that you could also verify that it can load a project file saved with 3.10.

Reply:
I'll make the modification. However, it appears like the backwards compatibility for stream is broken in master branching point itself. I'm able to load the project file - but the settings and preferences are lost.

Reply:
Uh oh, that doesn't sound good.  That'll need fixing regardless. :wink:

Reply:
Probably is due to differences in the type dumped in the stream. It was one of the problems with this binary format, which could be inadvertently modified. I'd like to take a look, but maybe it has a difficult solution. We might end up assuming that part of the compatibility of the 3.10 format will be lost in this version.